system/jlib/jstats.cpp

```diff
@@ -1836,7 +1829,7 @@ static void checkKind(StatisticKind kind)
     const char * shortName = queryStatisticName(kind);
     StringBuffer longName;
     queryLongStatisticName(longName, kind);
-    const char * tagName = queryTreeTag(kind);
+    const char * tagName __attribute__ ((unused)) = queryTreeTag(kind);
```

Comment:
Not really, queryTreeTag() has some dbgassertex's in it, perhaps that's why @ghalliday calls.